modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -225,12 +229,14 @@ For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
 This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `kernel` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see link:https://access.redhat.com/articles/7212[How does one set up a serial terminal and/or console in Red Hat Enterprise Linux?].
 ====
 +
+ifndef::openshift-origin[]
 [NOTE]
 ====
 To network boot the CoreOS `kernel` on `arm64` architecture, you need to use a version of iPXE build with the `IMAGE_GZIP` option enabled. See link:https://ipxe.org/buildcfg/image_gzip[`IMAGE_GZIP` option in iPXE].
 ====
+endif::openshift-origin[]
 endif::only-pxe[]
-
+ifndef::openshift-origin[]
```

Comment:
Just a writing guidelines thing. Can we combine these `ifndefs` since they are both not defining the same content, just for minimalism purposes.

`ifndef::openshift-origin[]`         -> `ifndef::only-pxe,openshift-origins[]` 
`ifndef::only-pxe[]`